### PR TITLE
Added a blog

### PR DESCRIPTION
**Pull request by andy6975**
Added a blog on how to build Tensorflow 1.14 with GPU and TensorRT support in Ubuntu 16.04

### CODE REVIEW

File: _posts/Build_TF_1.14_GPU_Ubuntu_16.0.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
+---
+layout: post
+author: "Shivam Chaudhary"
+author_url: https://github.com/andy6975
+title: "Building Tensorflow 1.14 GPU"
+subtitle: "How to build TF GPU in Ubuntu with TensorRT"
+bg_url: "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.tensorflow.org%2F&psig=AOvVaw1Gpx9AJEqAd0jPYRa57bWQ&ust=1588994195374000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCKjGke6mo-kCFQAAAAAdAAAAABAO"
```
Comment (aashish-ak) [inline _posts/Build_TF_1.14_GPU_Ubuntu_16.0.md:7]: This isn't opening, you can change it to [this](https://images.squarespace-cdn.com/content/v1/58cf449fe6f2e19aff94c85a/1526372369906-8QIF0U7SP7805EZWISNJ/ke17ZwdGBToddI8pDm48kNkMlmfixmshW_SeIs0xHj97gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0gdwqe6gATBfagbnBItpKKLXWV0U_MYk2qIuJRygurGwLtoEyxEa7HooetQk6_oPEw/Tensorflow+Banner+SS-01.png)

**Comment by aashish-ak** [comment]
Every code block should have python as the language as I'm currently working on fixing the CSS. Without setting python as the language, it'll not look good. Just add `python` after the opening backticks of the code block.

**Comment by aashish-ak** [comment]
Also change the name of the file to `2020-05-08-Build-TF-1.14-GPU-Ubuntu-16` otherwise Jekyll will not show the post on the page. This is the general format of naming the file of any post in jekyll.

**Comment by aashish-ak** [comment]
Also change the image code to `image alt ><` for the image to display properly.

**Comment by andy6975** [comment]
Made the necessary changes. Thanks for pointing them out! :)

**Pull request merged**